### PR TITLE
Field additions in proxy and authorization objects

### PR DESCRIPTION
**Pull request by floydtree**
1. Adding identifier fields in the proxy object
2. Adding policy identifer fields in the authorization object
3. consequent dictionary changes
Screen Shot 2022-09-20 at 4 39 40 PM
Screen Shot 2022-09-20 at 4 39 59 PM

### CODE REVIEW

File: dictionary.json [hunk 1]
```diff
@@ -2119,6 +2120,21 @@
       "description": "The number of bits per pixel.",
       "type": "integer_t"
     },
+    "policy_name": {
```
Comment (rroupski) [inline dictionary.json:2123]: We have a [policy](https://schema.ocsf.io/objects/dev/policy) object in the dev extensions, perhaps we can use it here?

File: dictionary.json [hunk 2]
```diff
@@ -265,7 +265,8 @@
     "authorization": {
       "caption": "Authorization Information",
       "description": "Authorization object provides details related to Authorization",
-      "type": "authorization"
+      "type": "authorization",
```
Comment (rroupski) [inline dictionary.json:268]: Rename the attribute as `authorizations`?

File: objects/authorization.json [hunk 3]
```diff
@@ -1,15 +1,21 @@
...
+  "description": "Details pertaining to Authorization Information",
+  "extends": "object",
+  "name": "authorization",
   "attributes": {
     "decision": {
       "description": "Authorization Result/outcome, e.g. allowed, denied",
       "requirement": "optional"
     },
-    "permission": {
+    "policy_name": {
```
Comment (rroupski) [inline objects/authorization.json:11]: Use the [policy](https://schema.ocsf.io/objects/dev/policy) object?

**Review by rroupski — CHANGES_REQUESTED** [review]

File: dictionary.json (hunk 1 above)
Comment (floydtree) [reply dictionary.json:2123]: Yeah good idea, I'll move that in, however the type_id in the policy object might need some rework. We can tackle improvement to the policy object in a separate PR

**Pull request merged**